Automattic/wp-calypso
# Big Sky: AI Powered pattern assembler

Related to seGwbA-bigskyai-p2

This introduces the `ai-assembler` flow where pattern assembler decisions are driven by a single prompt.

<img width="1213" alt="Zrzut ekranu 2023-09-21 o 16 48 20" src="https://github.com/Automattic/wp-calypso/assets/3775068/2a1c63ab-8149-4c0c-9035-dd0d72d3d39f">

## Proposed Changes

* Introduce the `ai-assembler` flow and `site-prompt` step to gather data for the AI call
* Pass the patterns selected from the backend to pattern assembler.

## Testing Instructions

- Get a site
- Run [calypso.live](https://calypso.live?image=registry.a8c.com/calypso/app:build-81588) to use prebuilt version or build from scratch locally
- Go to the `/setup/ai-assembler?siteSlug=YOURSLUG`
- Describe your site
- Submit
- Observe magic

## Pre-merge Checklist
